typescript/web/src/components/website/Features/Features.tsx

```diff
@@ -31,7 +31,7 @@ export const Features = () => {
             htmlHeight="320px"
             height={{ md: "320px" }}
             objectFit="cover"
-            src="/static/img/shot-elec.jpg"
+            src="/static/img/Shot-Elec 1.png"
```

Comment:
**suggestion**: Can we keep a convention for naming these files? 

⚠️ So we need to rename the file too!

```suggestion
            src="/static/img/shot-elec-1.png"
```